backend/API_standards.md

```diff
@@ -52,14 +52,32 @@ Nested resources will follow the same pattern:
 
 All endpoints should return a seamless response, independent of the endpoint action or expected result values.
 
+### Sessions
+
+To perform Login should be `POST /sessions` with email and password and it will returns an object with the auth information (token, expire date, etc) and the user information, everything inside of the payload key `data`.
```

Comment:
Maybe is worth to point that the `user` should correspond to the same User entity used across the other endpoints

Reply:
fixed